backend/e4h-services/ingestion-service/app/utils/project_service_client.py

```
@@ -291,6 +291,83 @@ def update_workflow(self, request_info: RequestInfo, project_id: str, action: st
+            }
+            
+            # Only add rowVersion if it exists in the source record
+            if row_version is not None:
+                project_facility_payload['rowVersion'] = row_version
+
+            update_payload = {
+                'RequestInfo': request_info.model_dump(by_alias=True, exclude_none=True),
+                'ProjectFacility': project_facility_payload
+            }
+            
+            update_response = requests.post(update_url, headers=update_headers, json=update_payload)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)